gemini/getting-started/intro_gemini_pro_python.ipynb

```diff
@@ -0,0 +1,652 @@
+{
```

Comment:
<p>The notebook is not model garden specific so it's not appreciate to use model garden in the title. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>